config/config.go

```diff
@@ -0,0 +1,60 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+// Package config gives constants and small functions that specify the REANA client configuration.
+package config
```

Comment:
We now have three package levels in this PR: (i) `cmd` with cobra stuff, (ii) `config` for abstracting hard-coded values, and (iii) `pkg` where all the displaying, formatting etc utility live (in their respective packages). The fact that a lot of business logic lives under `cmd` and not `pkg` creates a certain dissonance.

It may be good to find a way to have `cmd` stored under `pkg`, as other software projects do (e.g. kubectl, kind). In this way it'll be clear where the most important stuff is.  Also, `config` should then be moved there.

Note also that kubectl, kind and friends usually not use single `cmd` package for everything, but introduce packages for each command, which in our case would be `list`, `status`, etc. This would make sense because the whole reana-client-go is about CLI commands, so keeping cobra layer tiny, just for parsing CLI options etc, whilst having clearly visible business logic packages implementing each command business logic, would makes sense. From the pure package naming point of view, it seems advantageous to keep "cmd" tiny and place focus rather on the business logic itself.

Something we can think for later as the functionality grows.

Reply:
I moved `config` inside `pkg`. Separating the business logic from the cobra template seems out of reach of this PR but I could try creating different packages for each command. What do you think?

Reply:
Yes, we can go like this, and think about possible further changes as we shall be adding upload/secret/etc functionality.